docs/getting-started/about.rst

```diff
@@ -1,3 +1,7 @@
 ===============
 About PyMedPhys
-===============
\ No newline at end of file
+===============
+
+.. include:: ../../README.rst
+   :start-after: .. START_OF_ABOUT_IMPORT
+   :end-before: .. END_OF_ABOUT_IMPORT
```

Comment:
This is something quite neat about reST that I learnt tonight :)

Reply:
Very nice!